Assets/Scripts/Blockage.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Blockage : MonoBehaviour {
+
+    [SerializeField]
+    private PlayerCharacter player;
+
+    void Update ()
+    {
+        if (player.hasRing == true)
+        {
+            this.gameObject.SetActive(false);
+        }
+	}
```

Comment:
A little nitpick: maybe move the last } and line it up with the first { since I wasn’t sure what it was for at first.

Looks good!